unit-tests/func/FG/test-fg.py

```diff
@@ -0,0 +1,51 @@
+import pyrealsense2 as rs, test, ac
+
+dev = test.find_first_device_or_exit()
+depth_sensor = dev.first_depth_sensor()
+
+debug_sensor = rs.debug_streaming_sensor(depth_sensor)
```

Comment:
Exit (0) if `debug_sensor` is not valid -- such as on the D400

Reply:
Unless you look for L500 and then you can expect it

Reply:
Added find_devices_by_product_line_or_exit()